react/testing_5/src/app/screens/Dashboard/components/SignupResult/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+function SignupResult({ handleButtonClick }) {
```

Comment:
`onClick` ? Remember that `handle` names would refer to instance methods / well known handlers